src/igvfd/schemas/changelogs/mixins.md

```diff
@@ -1,5 +1,6 @@
 ## Changelog for *`mixins.json`*
 
+* Add `MPRAbase` to include `collections`.
```

Comment:
please update to "Add MPRAbase to collections.", not "Add `MPRAbase` to include `collections`"